languages/rust/oso/src/host/mod.rs

```diff
@@ -112,6 +112,17 @@ impl Host {
         }
     }
 
+    /// Register an MRO list for every registered class.
+    /// Since inheritance is not supported, all lists are empty.
+    pub fn register_mros(&self) -> crate::Result<()> {
+        for (name, _) in &self.classes {
+            if name != "oso::host::Class" {
+                self.polar.register_mro(Symbol(name.clone()), vec![])?;
```

Comment:
For languages that don't have inheritance, do we still need the ID of the class itself in the MRO list?

Reply:
We don't because the MRO is only checked if the class tags don't match. Putting the ID of the class in would make it more consistent but wouldn't do anything.

Reply:
Ok